### ISSUE TITLE
Major Stuttering when syncing large amount of songs

### ISSUE THREAD

**Issue by djdaquin** [JasonZigelbaum/jqbx-issues#35]
Not really upset by this or anything and somewhat expected it. I put my entire collection of saved music into the queue then tried to sync and the app is moving very slow.

With all that said you may want to consider some way of dealing with the large amount...also could be caused by a probable reddit hug as thats where I heard of this site.

**Comment by JasonZigelbaum** [comment]
How many songs do you have added? Just curious so I have an upper limit shoot for.

I'll start optimizing the frontend soon, I wanted to get the MVP out first which was tested to work well with ~200 tracks on the queue and starred playlists each.

**Comment by djdaquin** [comment]
So I normally listen to Spotify by just hitting "play all" on my saved songs. So I tried putting that into the queue playlist, which ended up being 1900 songs. A friend trued the same thing with 5k. Both caused the music to not be perfectly synced (the site was 20 seconds ahead of me, so the song would change early) and also made the site stutter...a lot. 

Idk how the site works when you have a large number of songs on the queue (it froze up enough that I wasn't able to scroll down) but maybe only take the top 200 songs and put them on the actual front end site? Someone who puts that many songs would have to give up fine control, but if I put 2000 songs into a playlist I'm not looking for fine control.

**Comment by JasonZigelbaum** [comment]
Yeah good idea I'll put a hard limit on it for now so it's less breakable. Ideally I'll be able to handle that many tracks without an issue though in the future with some added infinite scrolling.

I'll keep you posted. Thanks again

**Comment by djdaquin** [comment]
If you are targeting a number to handle completely, Spotify maxes playlists at 10,000.
